backend/src/middleware/notifications/notificationsMiddleware.js

```diff
@@ -28,14 +30,16 @@ const postAuthorOfComment = async (commentId, { context }) => {
   try {
     postAuthorId = await session.readTransaction(transaction => {
       return transaction.run(
-        ` 
+        `
           MATCH (author:User)-[:WROTE]->(:Post)<-[:COMMENTS]-(:Comment { id: $commentId })
           RETURN author { .id } as authorId
         `,
         { commentId },
       )
     })
     return postAuthorId.records.map(record => record.get('authorId'))
+  } catch (error) {
+    debug(error)
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-23T18:27:01Z" title="Thursday, January 23rd 2020, 7:27:01 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -36,6 +38,8 @@ const postAuthorOfComment = async (commentId, { context }) => {
       )
     })
     return postAuthorId.records.map(record => record.get('authorId'))
+  } catch (error) {
+    debug(error)
```


like that you are adding these now.... I will try to get in the habit of doing this as well!